civis/polling.py

```diff
@@ -8,6 +8,37 @@
 _DEFAULT_POLLING_INTERVAL = 15
 
 
+class ResultPollingThread(threading.Thread):
```

Comment:
I left this public, but maybe it ended up specific enough that it should be private?